src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -182,4 +182,5 @@
   register: result
   until: not result.failed or 'process not presently configured with monit' not in result.msg
   retries: 5
-  delay: 20
\ No newline at end of file
+  delay: 20
+  
```

Comment:
Does github also not like an extra newline, or is there other whitespace on that line?

Reply:
Looking into this now. Thought my IDE was configured appropriately.